loleaflet/src/control/Control.UIManager.js

```diff
@@ -75,14 +75,16 @@ L.Control.UIManager = L.Control.extend({
 		this.map.addControl(L.control.infobar());
 		this.map.addControl(L.control.userList());
 
-		var map = this.map;
-
 		var openBusyPopup = function(label) {
 			var json = {
 				id: 'busypopup',
 				jsontype: 'dialog',
 				type: 'modalpopup',
 				children: [
+					{
+						id: 'jsdialog-overlay',
+						type: 'overlay'
+					},
```

Comment:
Unfortunately the overlay cannot really be inside of the #busypopup otherwise it becomes above the dialog and ends up covering. The overlay should be added at the some level as #busypopup 
Something like this (of course it doesn't work , just an idea) ? :
```js
			var json = {
				children: [
					{
						id: 'jsdialog-overlay',
						type: 'overlay'
					},
					{
						id: 'busypopup',
						jsontype: 'dialog',
						type: 'modalpopup',
						children: [
							{
								id: 'busycontainer',
								type: 'container',
								vertical: 'true',
								children: [
									{id: 'busyspinner', type: 'spinnerimg'},
									{id: 'busylabel', type: 'fixedtext', text: label}
								]
							}
						]
					}
				]
			};
```